sdk/storage/Azure.Storage.Blobs/tests/BlobTestBase.cs

```diff
@@ -16,8 +16,10 @@
 
 namespace Azure.Storage.Test.Shared
 {
+    [ClientTestFixture(BlobClientOptions.ServiceVersion.V2019_02_02)]
```

Comment:
Add a second service version here when available. Follow the same pattern with other ***TestBase classes.